src/util/helpers.js

```diff
@@ -85,13 +90,13 @@ const devConstants = {
 
 // NEW: If you have checked the "Using Dev Lattice" box in settings, the constants
 // are swapped out here
-const localSettings = getLocalStorageSettings();
+const localSettings = localStorage.getSettings();
 if (localSettings.devLattice) {
     Object.keys(devConstants).forEach((key) => {
         constants[key] = devConstants[key];
     })
 }
-exports.constants = constants;
```

Comment:
Wait... we don't use these constants anywhere else?

Reply:
They're exported above. It should be the same behavior as before.